src/chrome/content/rules/University-of-Exeter.xml

```diff
@@ -26,6 +30,10 @@
 	<securecookie host="^(?:.*\.)?exeter\.ac\.uk$" name=".+" />
 
 
+	<!-- Cert only matches empslocal.ex.ac.uk -->
+	<rule from="^http://empslocal\.ex\.ac\.uk/"
+		to="https://empslocal.ex.ac.uk/" />
+
 	<rule from="^http://newton\.ex\.ac\.uk/"
 		to="https://newton.ex.ac.uk/" />
```

Comment:
Ah, so presumably newton.exeter.ac.uk should be handled by the subsequent rule, since if it should always be rewritten to exeter, it's not a special case.